### DISCUSSION TITLE
Domain redirection

### DISCUSSION THREAD

**Discussion by Abhinav0002** [vercel/community#3503] [Help] [answered]
### Question

I had used a custom domain for my project, but I have deleted the project long back.
Now when I am using the domain for deployment on another service, it is still redirecting to a Vercel deployment, how to resolve this ?

**Comment by amyegan** [comment, marked as answer]
Hey @Abhinav0002. If you want the domain to point to a different Vercel project, you can manage that in project settings. If you want the domain to point to a different site or service, then you'll need to go to your domain registrar and change the configuration there.
